### ISSUE TITLE
Interface Builder - Not able to create outlets

### ISSUE THREAD

**Issue by alexkblount** [xamarin/xamarin-macios#4619]
### Steps to Reproduce

1. Create new Xamarin iOS Application
2. Right Click Storyboard
3. Open in Interface Builder
4. Create a control in your ViewController
5. Connect this to the code behind using CMD Drag

### Expected Behavior

Outlet should be created in the header file of the ViewController

### Actual Behavior

Error: Could not insert new outlet connection: Could not find any information for the class ViewController

### Environment

=== Visual Studio Enterprise 2017 for Mac ===

Version 7.5.4 (build 3)
Installation UUID: 59e8a0c2-a674-42a6-b852-843de896e0eb
Runtime:
	Mono 5.10.1.57 (2017-12/ea8a24b1bbf) (64-bit)
	GTK+ 2.24.23 (Raleigh theme)
	Xamarin.Mac 4.4.1.178 (master / eeaeb7e6)

	Package version: 510010057

=== NuGet ===

Version: 4.3.1.4445

=== .NET Core ===

Runtime: /usr/local/share/dotnet/dotnet
Runtime Version: 2.1.1
SDK: /usr/local/share/dotnet/sdk/2.1.301/Sdks
SDK Version: 2.1.301
MSBuild SDKs: /Library/Frameworks/Mono.framework/Versions/5.10.1/lib/mono/msbuild/15.0/bin/Sdks

=== Xamarin.Profiler ===

Version: 1.6.2
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

=== Xamarin.Android ===

Version: 8.3.3.2 (Visual Studio Enterprise)
Android SDK: /Users/blounty/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		7.1 (API level 25)
		8.1 (API level 27)

SDK Tools Version: 26.1.1
SDK Platform Tools Version: 28.0.0
SDK Build Tools Version: 28.0.2

Java SDK: /usr
java version "1.8.0_131"
Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)

Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

=== Apple Developer Tools ===

Xcode 9.4.1 (14161)
Build 9F2000

=== Xamarin.Mac ===

Version: 4.4.1.193 (Visual Studio Enterprise)

=== Xamarin.iOS ===

Version: 11.12.0.4 (Visual Studio Enterprise)
Hash: 64fece5f
Branch: d15-7
Build date: 2018-05-29 20:00:44-0400

=== Xamarin Inspector ===

Version: 1.4.0
Hash: b3f92f9
Branch: master
Build date: Fri, 19 Jan 2018 22:00:34 GMT
Client compatibility: 1

=== Build Information ===

Release ID: 705040003
Git revision: 6ae731889c896d6733efb8ff5117f5bf5b17b509
Build date: 2018-07-19 13:07:45-04
Xamarin addins: 417fed09624e1e1f76ab0a11e8b97ffd8bbf91e1
Build lane: monodevelop-lion-d15-7

=== Operating System ===

Mac OS X 10.13.6
Darwin 17.7.0 Darwin Kernel Version 17.7.0
    Thu Jun 21 22:53:14 PDT 2018
    root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64

<img width="1513" alt="screen shot 2018-08-14 at 11 01 05 am" src="https://user-images.githubusercontent.com/30159621/44109241-63689946-9fb1-11e8-94c7-613b646933a7.png">

> VS bug [#664435](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/664435)

**Comment by jstedfast** [comment]
The problem is that until Xcode indexes the exported Objective-C project, a user cannot connect any new Outlets or Actions.

What you need to do is wait until you see Xcode index the project in the status display at the top of the Xcode window (You'll see text like "Indexing project..." and a progress bar).

Xcode has apparently gotten slower over the releases and it now takes a few minutes before it indexes the project :(

I'm not sure what VSMac can do to force Xcode to index sooner...

**Comment by alexkblount** [comment]
I don't think this is an issue with indexing, Xcode actually shows no sign of indexing and no matter how long it is left does not allow the creation of outlets.


![vs4m-xcode](https://user-images.githubusercontent.com/30159621/45386479-4aee0680-b5c8-11e8-8204-75f41b4e4966.gif)

**Comment by jstedfast** [comment]
You might need to do something to trigger the indexing, but it *is* indexing.

Try adding a source file to the project or something.

**Comment by alexkblount** [comment]
I added a new file, no indexing begins, waited and I am still not able to create outlets.

<img width="1399" alt="screen shot 2018-09-11 at 2 10 44 pm" src="https://user-images.githubusercontent.com/30159621/45388041-ac17d900-b5cc-11e8-878f-b778372d6cc9.png">

**Comment by jstedfast** [comment]
I'm not sure how I triggered Xcode to index, but I did (all I did was switch away to Chrome to search Stackoverflow.com how to trigger indexing in Xcode), when I switched back to Xcode it was indexing and then it worked.
